server/datamodel.prisma

```diff
@@ -1,4 +1,28 @@
 type User {
   id: ID! @id
   name: String!
+  password: String!
+  email: String!
+  phone: Int
+  emergency: Int
+  booked: [Booking]!
+  favourite: [String]
 }
+type Campsite {
+  id: ID! @id
+  campname: String!
+  price: Int!
+  description: String
+  facilities: [String]
+  details: [String]
+  reviews: [String]
+  location: [Float!]!
```

Comment:
should be lat, long?

Reply:
It was going to be an array of two indexes for lat and long but I could separate it.  Which is better?